### DISCUSSION TITLE
Community driven icon

### DISCUSSION THREAD

**Discussion by aeharding** [aeharding/voyager#262] [General]
This discussion is reserved for figuring out how best to adopt a community-driven icon for wefwef :)

**Comment by fer0n** [comment]
I'd say make an announcement that you're looking for an icon with a deadline in maybe 1 or 2 weeks, collect them all and put them to a vote.

If it's possible to change the icon, we could have icon packs later down the line. Don't know about Android, but I'm pretty sure on iOS you'd have to delete and re-add the app, so it sadly wouldn't be super easy from a user perspective.

**Reply by fer0n** [reply]
And depending on how many suggestions there are make a pre-selection yourself or only pick the ones you'd be happy with

**Comment by Kinyuu** [comment]
Would it be possible to add the Material You design of Android to the app icon?

**Reply by kra-mo** [reply]
AFAIK the PWA spec doesn't support it (yet).

**Comment by fer0n** [comment]
For voting, I'd say use something that lets users sort/enumerate the icons by priority. If that's too much effort go for classic up/down votes, but definitely not a "pick only one" vote (there's a video by CGP Grey explaining why that's a bad idea: https://youtu.be/3Y3jE3B8HsE).

There's also the question of how to handle variants of the same icon with e.g. different color themes. If they're allowed are they voted on separately or in a second voting round? (Simultaneously isn't a good idea if you can only pick one)

**Comment by fer0n** [comment]
Seems to be coming along quite nicely 🙂
@aeharding I'd suggest for the final vote to primarily (or at least prominently) show the icons on an actual icon size (maybe side-by-side). It's difficult to judge how a full blown icon will look as an actual app icon.
